sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -2239,6 +2239,11 @@ pi_result piDeviceRelease(pi_device Device) {
 pi_result piDeviceGetInfo(pi_device Device, pi_device_info ParamName,
                           size_t ParamValueSize, void *ParamValue,
                           size_t *ParamValueSizeRet) {
+  if (PI_DEVICE_INFO_BACKEND_VERSION == ParamName) {
+    // TODO: return some meaningful for backend_version below
+    ReturnHelper ReturnValue(ParamValueSize, ParamValue, ParamValueSizeRet);
+    return ReturnValue("");
+  }
   return pi2ur::piDeviceGetInfo(Device, ParamName, ParamValueSize, ParamValue,
                                 ParamValueSizeRet);
 }
```

Comment:
this code is going away (replaced bv the code in `pi2ur::piDeviceGetInfo`) so we should not add new code here. can we just fall through to `pi2ur::piDeviceGetInfo` and have it return an error of unrecognized parameter?

Then we can either throw the error to the user (since we told them not to use it in the spec), handle the error in SYCL RT and continue gracefully.

Reply:
Spec says `implementations are encouraged to return the empty string`, not an error. Am I missed something?
And, so is it ok to move this code on lines 2242-2246 to pi2ur::piDeviceGetInfo?

Reply:
yes, either move to `plugins/unified_runtime` or handle error (turn into an empty string) within SYCL RT

Reply:
Thanks, CUDA plugin already returns some meaningful value, so to not over-complicate logic, UR plugin returns empty string now.